code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -5,13 +5,69 @@
 
 /obj/structure/closet/crate/necropolis
 	name = "necropolis chest"
-	desc = "It's watching you closely."
+	desc = "It's watching you suspiciously."
 	icon_state = "necrocrate"
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 	door_anim_time = 0
```

Comment:
does this actually start locked?

Reply:
non-tendril necropolis chests, like megafauna necropolis chests, start unlocked and (should) have the same behavior as before... at least that was what the author intended

Reply:
yes